### PR TITLE
Added missing callback to the AdController

### PR DESCRIPTION
**Pull request by thomas-james-1986**
This sets the callback property on the `AdController` for the `VideoViewController` to ensure that events are dispatched back to the `VideoAd` callback.

https://user-images.githubusercontent.com/30452349/179216388-afd352b3-6cfa-4cdd-a7ac-1ff603d7efc9.mov

### CODE REVIEW

**Review by tashmulla — APPROVED** [review]
Nice! 👌 Just wondering, any tests we can add for callbacks? Either in this ticket or a following one 🙂

File: Pod/Classes/UI/Video/VideoViewController.swift [hunk 1]
```diff
@@ -25,6 +25,7 @@ import UIKit
         videoEvents = VideoEvents(adResponse)
         super.init(nibName: nil, bundle: nil)
         self.controller.adResponse = adResponse
+        self.controller.callback = callback
```
Comment (gnhn) [inline Pod/Classes/UI/Video/VideoViewController.swift:28]: `callback` property on the same file seems redundant. Could you confirm and remove it please?

**Comment by thomas-james-1986** [comment]
@tashmulla yes I can add some unit tests for the callback

File: Example/SuperAwesomeExampleTests/UI/Common/AdControllerTests.swift [hunk 2]
```diff
@@ -38,4 +38,128 @@ class AdControllerTests: XCTestCase {
         showPadlockTest(showPadlock: true, ksfUrl: nil, expectedResult: true)
         showPadlockTest(showPadlock: false, ksfUrl: "http", expectedResult: false)
     }
+
+    // Callbacks
```
Comment (tashmulla) [inline Example/SuperAwesomeExampleTests/UI/Common/AdControllerTests.swift:42]: 😍 👏

**Review by tashmulla — APPROVED** [review]

**Pull request merged**